blocks/init/src/Blocks/components/button/button.php

```diff
@@ -62,7 +61,7 @@
 ?>
 
 <?php
-	echo Components::outputCssVariables($attributes, $manifest, $unique, $globalManifest);
+	echo Components::outputCssVariables($attributes, $manifest, $unique);
```

Comment:
Shouldn't these be `Helpers::` ?

Reply:
yes, it should, but I will make this a separate PR because there are a lot of changes.